src/rex.mart/src/rex/mart/ctl.py

```diff
@@ -127,6 +127,8 @@ def __call__(self):
                         )
                     )
 
+                    get_sentry().captureException(sync=True)
```

Comment:
I'd move this below to the `else:` branch (to be created) of the `if entry.halt_on_failure:`. Otherwise, I think, the exception will be reported twice if `entry.halt_on_failure` is `True`.

Reply:
I agree that's how it looks like it would work, but we tried using `halt_on_failure` at one point and we still didn't see mart errors reported in Sentry...

https://github.com/prometheusresearch/lfa-system/commit/67adc5a723b39b099b9c384deae967e969d79864#diff-76a52879af8738f7286ad8218203ffd074244ac6a198c1b8db18609e75f9f807